### ISSUE TITLE
Automate Inserting Stats

### ISSUE THREAD

**Issue by objectmethod** [kinabalu/obsidian-oura-plugin#21]
Somehow automate inserting of data would be a great feature addition, I believe.

Maybe, when you create a new daily note, your daily note template file would have a script like 

```oura
insertStats('YYYY-MM-DD')
```

¯\_(ツ)_/¯

**Comment by pedramamini** [comment]
Yes, seconded! Additionally if we can specify the YYYY-MM-DD then we can backfill for an entire date range.

**Comment by finn-matti** [comment]
If you use the [templater](https://github.com/SilentVoid13/Templater) plugin, you can use this in your template to insert those stats:

```
<% tp.app.commands.executeCommandById('obsidian-oura-plugin:insert-oura-ring-stats') %>
```

The command itself doesn't seem to allow extra parameters, but at least you can add the oura stats automatically for the current day in this way.

**Comment by pedramamini** [comment]
That appears to simply insert "true"

**Comment by objectmethod** [comment]
> If you use the [templater](https://github.com/SilentVoid13/Templater) plugin, you can use this in your template to insert those stats:
> 
> ```
> <% tp.app.commands.executeCommandById('obsidian-oura-plugin:insert-oura-ring-stats') %>
> ```
> 
> The command itself doesn't seem to allow extra parameters, but at least you can add the oura stats automatically for the current day in this way.

That's incredible! Thanks!

**Comment by kinabalu** [comment]
Have integrated the PR into 0.2.6 and pushed it out, I know it's been forever but latest is up there. Cheers!
